Tests/FluentAssertions.Specs/Primitives/DateOnlyAssertionSpecs.cs

```diff
@@ -0,0 +1,863 @@
+using System;
+using Xunit;
+using Xunit.Sdk;
+
+#if NET6_0_OR_GREATER
+
+namespace FluentAssertions.Specs.Primitives
+{
+    public class DateOnlyAssertionSpecs
+    {
+        #region (Not) Be
+
+        [Fact]
+        public void Should_succeed_when_asserting_dateonly_value_is_equal_to_the_same_value()
+        {
+            // Arrange
+            DateOnly dateOnly = new(2016, 06, 04);
+            DateOnly sameDateOnly = new(2016, 06, 04);
+
+            // Act
+            Action act = () => dateOnly.Should().Be(sameDateOnly);
+
+            // Assert
+            act.Should().NotThrow();
```

Comment:
🔧 We prefer to not use an action and `NotThrow` if an exception is not excepted. You just call `dateOnly.Should().Be(sameDateOnly)` with a comment statement `// Act/Assert`